src/communication/WebSocketWrapper.ts

```diff
@@ -0,0 +1,13 @@
+import {fromEvent as observableFromEvent,  Observable } from "rxjs";
```

Comment:
all that to make it testable?

Reply:
why not replacing the websocket by a subject (http://reactivex.io/rxjs/manual/overview.html#subject).
much more standard, and will make it easier to update later.

Reply:
I don't think that it's possible to send messages with a Subject no ?
Here we also need the send function.

Reply:
With a next()?
I'm almost sure that's what we used in the py API

Reply:
https://github.com/xcomponent/ReactiveXComponent.py/blob/e8c661d76c4bc58cdd606e6d0762b51ec879bdf9/reactivexcomponent/reactivexcomponent/communication/xc_session.py